### DISCUSSION TITLE
Where can I find libAvalonia.Native.OSX.dylib?

### DISCUSSION THREAD

**Discussion by wj8400684** [AvaloniaUI/Avalonia#9931] [Q&A] [answered]
Downloading the latest source code does not work on the Mac OS platform! Prompt that this file cannot be found！

system os: mac os 13
cpu：m1
ide：rider

**Comment by wj8400684** [comment, marked as answer]
Solved!

**Comment by dfernandezramos** [comment]
Just as an advice for people that may have your same issue, this can be solved by following the guide at: https://docs.avaloniaui.net/guides/developer-guides/build-avalonia-from-source

Running the `./build.sh --target CompileNative` command at the root path of the repository should generate the file and then you can proceed building the app as indicated 😄

**Comment by NicholasFreeman** [comment]
The above build-avalonia-from-source link is dead. This works: https://github.com/AvaloniaUI/Avalonia/blob/master/docs/build.md
